src/partials/landing.html

```diff
@@ -1,17 +1,14 @@
 <div id="landing" class="grid-x align-center align-middle">
   <div class="cell medium-6 large-8 text-center">
-    <h1>
-      DVC<br>
-      HACKS
-    </h1>
-    <br>
-    <p>
+    <img src="img/dvchacks-logo-white.png" class="logo">
+
+    <p class="caption">
       Come hack with us and compete for prizes!
     </p>
     <div class="divider"></div>
     <div class="event-info">
       <p>April 13-14</p>
-      <p>12:00AM - 12:00AM</p>
+      <p>9:00AM - 8:00PM & 9:00AM - 2:00PM </p>
```

Comment:
Might be a good idea to indicate which day is for which time, like "Saturday on 9 AM - 8 PM" and "Sunday on 9 AM - 2 PM"

Reply:
I added it at first but it looked a bit too long. I'll put it back and we'll see how it looks.